src/services/extensions/links.test.ts

```diff
@@ -669,3 +810,7 @@ function getPath(options: {
     options.expectedFieldsUrlString ?? ''
   }${options.expectedLevelsFilterUrlString ?? ''}`;
 }
+
+function addAdHocFilterUserInputPrefixAndValueLabel(value: string, valueLabel = value) {
```

Comment:
This seems to be always called with a single argument? I.e. no valueLabel. Also not a huge fan of this function's name.